### DISCUSSION TITLE
[SOLVED] G7 connectivity issue

### DISCUSSION THREAD

**Discussion by polloze** [NightscoutFoundation/xDrip#3075] [Bug reports]
Well, it all started with my old G7 sensor that I accidentally ripped of while I took a shower. So I put a new sensor on, got a pairing request that I accepted. I might have a mistake afterwards as I was confused if I got the right pairing. I somehow thought it might be the old sensor and disconnected it from the bluetooth devices.

Since then I can't get the new sensor working though I saved the correct Transmitter ID. Checked the xdrip settings (Navid told me I had some incorrect settings which I changed - nontheless it did work before with those incorrect settings). But nothing seems to help to get sensor working with xdrip.
At least I can get the current sensor working with the original app and use companion mode.

I already uploaded my Xdrip log file. Hopefully it contains all the data that's needed.

**Comment by rudysroost** [comment]
I had the same issue for many weeks, I finally decided to uninstall xDrip completely and then reinstall. Doing so resolved the issue for me.

**Comment by polloze** [comment]
Thanks. I'll try that on sunday when I have lots of time. Do I need to do any backups?

**Comment by rudysroost** [comment]
> Thanks. I'll try that on sunday when I have lots of time. Do I need to do any backups?
Only if you want to back up your earlier readings, if not, they will be gone. In my case, they were on Nightscout, so I wasn't concerned.

**Comment by Navid200** [comment]
@polloze According to your post on facebook, this has been resolved.
There are individuals who don't have facebook accounts.  To make this a self-contained thread, it would be nice if you could confirm that the problem is resolved and if you like to add anything for anyone else who may find this thread through search who may be experiencing a similar problem.
Thanks

**Reply by polloze** [reply]
@Navid200 Sure, I wanna share this with others:
Basically I haven't done anything than waiting that my last sensor which didn't want to work with Xdrip ran off (G7 app and xdrip with companion mode). Well, you recommended to change some Xdrip settings which I've done. As it worked before this wasn't part of the problem.

So, I waited til my last sensor was done after it's 10 days. Then I did clean my skin. Waited for about 5 minutes. Set Xdrip back to G7 mode, changed the Transmitter ID, went to System status and got the pairing request as it supposed to be. And yes, the new sensor works fine as I got already my first readings.

Conclusion: I really did mess up something with the old sensor and the pairing process. 😃
